### ISSUE TITLE
Xamarin GTK on raspberry Pi

### ISSUE THREAD

**Issue by Azmarel** [jsuarezruiz/forms-gtk-progress#17]
Hi
How do you compile the application from Visual Studio for arm platform? I can launch the compiled applications succesfully on normal linux computer but i'm having difficulties running the apps on Raspberry Pi with Raspbian,
I get error:
 cannot execute binary file: Exec format error

**Comment by thenderson21** [comment]
Have you installed mono on the raspberry pi?

**Comment by Azmarel** [comment]
Yes everything is installed mono-complete and gtk-sharp2, i think the problem is that it is compiled as x86 and raspberry pi required arm compilation.

**Comment by thenderson21** [comment]
I have not hat to specify an architecture, with any of my Xamarin.forms PI projects when during compilation. I usually see the "Exec format error" when trying to run executable directly without calling mono first, ie. mono SomeApp.exe.

**Comment by duanenewman** [comment]
I've ran it many times on my Raspberry Pi. Architecture set to AnyCpu. you can see a reference project that runs on the RPi on my CADI repo: https://github.com/duanenewman/cadi
